skyportal/tests/api/test_sources.py

```diff
@@ -2326,3 +2326,93 @@ def test_add_and_delete_source_label(upload_data_token, view_only_token, public_
     assert data["data"]["id"] == obj_id
 
     assert len(data["data"]["labellers"]) == 0
+
+
+def test_copy_photometry_sources(
+    public_group, upload_data_token, ztf_camera, view_only_token
+):
+    obj_id1 = str(uuid.uuid4())
+    obj_id2 = str(uuid.uuid4())
+    ra = 200.0 * np.random.random()
+    dec = 90.0 * np.random.random()
+    status, data = api(
+        "POST",
+        "sources",
+        data={
+            "id": obj_id1,
+            "ra": ra,
+            "dec": dec,
+            "redshift": 3,
+            "group_ids": [public_group.id],
+        },
+        token=upload_data_token,
+    )
+    assert status == 200
+    status, data = api(
+        "POST",
+        "sources",
+        data={
+            "id": obj_id2,
+            "ra": ra + 0.0001,
+            "dec": dec + 0.0005,
+            "redshift": 3,
+            "group_ids": [public_group.id],
+        },
+        token=upload_data_token,
+    )
+    assert status == 200
+    status, data = api(
+        "POST",
+        "photometry",
+        data={
+            "obj_id": obj_id1,
+            "mjd": 59801.4,
+            "instrument_id": ztf_camera.id,
+            "filter": "ztfg",
+            "group_ids": [public_group.id],
+            "mag": 12.4,
+            "magerr": 0.3,
+            "limiting_mag": 22,
+            "magsys": "ab",
+        },
+        token=upload_data_token,
+    )
+    assert status == 200
+    status, data = api(
+        "POST",
+        "photometry",
+        data={
+            "obj_id": obj_id2,
+            "mjd": 59801.3,
+            "instrument_id": ztf_camera.id,
+            "filter": "ztfg",
+            "group_ids": [public_group.id],
+            "mag": 12.4,
+            "magerr": 0.3,
+            "limiting_mag": 22,
+            "magsys": "ab",
+        },
+        token=upload_data_token,
+    )
+    assert status == 200
+
+    status, data = api(
+        "POST",
+        f"sources/{obj_id1}/copy",
```

Comment:
I feel like maybe the URL path should say `copy_photometry` just to be more specific.

Reply:
yeah this was cause it was using current_user instead of associated_user_or_object. I will strip that out.

Reply:
yes this is fixed.